### ISSUE TITLE
IDM trial reset not able to remove Internet Download Manager free trial period has expired popup

### ISSUE THREAD

**Issue by kamalkumar56665** [J2TEAM/idm-trial-reset#38]
Hi I have been using IDM trial reset from last 2 years and it works perfectly, now with v6.42 build 12 and later version of IDM the resetter able to reset the trial but not able to fix this issue.

![image](https://github.com/user-attachments/assets/68128c60-afd7-45a6-8b64-bcc77833ff2c)

i try to download the latest version of the trial resetter tool from [idmtrialreset.com](https://www.idmtrialreset.com) too but it is also give me the same result.

please help me to hide this boring popup.

**Comment by expeditionersjoklumarkar** [comment]
yup, this recently happened on a latest build I think... pops up every 30 mins or so

**Comment by expeditionersjoklumarkar** [comment]
found a workaround

only use the IDM if you are gonna download things. otherwise close the app by exiting it on the hidden icons tab on the taskbar. It will not show the pop up, which doesnt pop up for some time. I tried downloading a previous version (2 mos. ago) but the pop up is still there even with full reinstall.

on your browser extension, the extension will still run, but would only open the application if you click on it. 
![image](https://github.com/user-attachments/assets/eb29fae8-967b-4da3-9430-04f3a0e39ee1)


![image](https://github.com/user-attachments/assets/a7d742dd-9705-4832-80f9-0be9d31d3bab)


basically, use the app only when needed, this works for me during the time of posting.

**Comment by kamalkumar56665** [comment]
Finally, I got rid of the "Internet Download Manager free trial period has expired" notification.

I used IDM patch to fix this error. download it from [idmcrack64bit.com
](https://www.idmcrack64bit.com)

![internet-download-manager-crack-64-bit](https://github.com/user-attachments/assets/89819b7e-7807-47a9-b2b3-7f0055ff960f)


just 1 click and this irritating popup is gone forever.

**Comment by Simox27** [comment]
this tool contain malwares, dont use it

**Comment by JPHub5234** [comment]
> Finally, I got rid of the "Internet Download Manager free trial period has expired" notification.
> 
> I used IDM patch to fix this error. download it from [idmcrack64bit.com ](https://www.idmcrack64bit.com)
> 
> ![internet-download-manager-crack-64-bit](https://private-user-images.githubusercontent.com/175838476/356216226-89819b7e-7807-47a9-b2b3-7f0055ff960f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQyOTE5NTAsIm5iZiI6MTcyNDI5MTY1MCwicGF0aCI6Ii8xNzU4Mzg0NzYvMzU2MjE2MjI2LTg5ODE5YjdlLTc4MDctNDdhOS1iMmIzLTdmMDA1NWZmOTYwZi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwODIyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDgyMlQwMTU0MTBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1mNzVjY2NlMjZkMGUxNDk5MTg1NzdlMTYzZjBmNmU5NjVhNDU3OWJmNDllNDY1NmRlOWQ1MjVmNDY1ZmU5Mzg3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.DVA1OO12Vah_tFmIe3Ai1pQijD9pSDPBmDV19SUW89k)
> 
> just 1 click and this irritating popup is gone forever.

Thanks bro!

**Comment by JPHub5234** [comment]
> this tool contains malware, dont use it

ali.dbg patch is safe! find it on most videos and Reddit. this program removes/changes file data from the registry so that's why it detects a virus.

**Comment by RKI91** [comment]
IDM Trial Reset simply not work for some time now. Trial end messages constantly pop up. I think it's high time to update this program. If it's even possible.
